app/pages/covid19/components/covid19-solution-card.test.js

```diff
@@ -0,0 +1,197 @@
+import { createTestHarness } from '../../../test-utils/testHarness';
+
+const setup = {
+  component: {
+    name: 'covid19SolutionCard',
+    path: 'pages/covid19/components/covid19-solution-card.njk',
+  },
+};
+
+describe('covid19-solution-card', () => {
+  it('should render the covid19 tag', createTestHarness(setup, (harness) => {
+    const context = {
+      params: {
+        solution: {},
+      },
+    };
+
+    harness.request(context, ($) => {
+      const covid19SolutionIndicator = $('[data-test-id="solution-card-covid19-tag"]');
+      expect(covid19SolutionIndicator.length).toEqual(1);
+      expect(covid19SolutionIndicator.text().trim()).toEqual('Coronavirus');
+    });
+  }));
+
+  it('should render the view this solution link', createTestHarness(setup, (harness) => {
+    const context = {
+      params: {
+        solution: {
+          id: 'S1',
+          viewSolutionUrl: '/solutions/covid19/S1',
+        },
+      },
+    };
+
+    harness.request(context, ($) => {
+      const viewSolutionLink = $('[data-test-id="solution-card-view-link"]');
+      expect(viewSolutionLink.length).toEqual(1);
+      expect(viewSolutionLink.text().trim()).toEqual('View this solution');
+      expect(viewSolutionLink.find('a').attr('href')).toEqual(`/solutions/covid19/${context.params.solution.id}`);
```

Comment:
is that just the `viewSolutionUrl` from the context?

Reply:
Yeah, ill get it from there instead